airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/state/MemoryManager.kt

```diff
@@ -60,6 +61,10 @@ interface AvailableMemoryProvider {
     val availableMemoryBytes: Long
 }
 
+// Since this is obviously the default implementation outside of unit tests
+// consider using @DefaultImplementation instead of @Secondary.
```

Comment:
Where has this annotation been all my life? Thanks, I'll use it.

Reply:
It might be a recent addition to Micronaut, actually.